step1_data_preparation/clean_business.py

```diff
@@ -0,0 +1,50 @@
+import ujson
+import pandas as pd
+
+business_ids = []
```

Comment:
You should be able to instantiate a pandas dataframe from a single list of dictionaries. No need for a list for each column.